app/(auth)/(tabs)/_layout.tsx

```diff
@@ -1,10 +1,31 @@
-import { TabBarIcon } from "@/components/navigation/TabBarIcon";
-import { Colors } from "@/constants/Colors";
-import { BlurView } from "expo-blur";
 import * as NavigationBar from "expo-navigation-bar";
-import { Tabs } from "expo-router";
 import React, { useEffect } from "react";
-import { Platform, StyleSheet } from "react-native";
+import { Platform, View } from "react-native";
+
+import { withLayoutContext } from "expo-router";
+
+import {
+  createNativeBottomTabNavigator,
+  NativeBottomTabNavigationEventMap,
+} from "react-native-bottom-tabs/react-navigation";
+
+const { Navigator } = createNativeBottomTabNavigator();
+
+import { BottomTabNavigationOptions } from "@react-navigation/bottom-tabs";
```

Comment:
Small nit: you should import NativeBottomTabNavigationOptions from `react-native-bottom-tabs/react-navigation` this will give you the correct types! 👍🏻